client/components/jetpack/card/jetpack-product-card-alt/index.tsx

```diff
@@ -50,7 +50,8 @@ type OwnProps = {
 	isOwned?: boolean;
 	isDeprecated?: boolean;
 	expiryDate?: Moment;
-	hidePrice?: boolean;
```

Comment:
In this new version of the Plans grid, all products feature their prices, so we no longer need this property.